hypha/static_src/sass/components/_nav.scss

```diff
@@ -1,64 +1,18 @@
 .nav {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-    width: 100%;
-    height: 100%;
-    font-weight: $weight--semibold;
-
-    @include media-query(tablet-portrait) {
-        flex-direction: row;
-        padding-block-start: 0;
-    }
-
-    &--primary {
-        text-transform: uppercase;
-
-        @include media-query(tablet-portrait) {
-            flex-direction: row;
-            justify-content: center;
-            gap: 2rem;
-            margin-block-start: 0;
-            text-transform: none;
-        }
-
-        @include media-query(tablet-landscape) {
-            gap: 3rem;
-        }
-    }
-
-    &__item {
-        @include media-query(tablet-portrait) {
-            &:last-child {
-                margin-inline-end: 0;
-            }
-        }
-    }
-
     &__link {
         @extend %h5;
+        display: flex;
         position: relative;
-        display: inline-block;
-        padding: 20px 10px;
-        color: $color--white;
+        padding-block: 0.5rem;
+        color: $color--default;
         transition: color $transition;
 
         &:focus,
         &:hover {
             color: $color--light-blue;
         }
 
-        @include media-query(tablet-portrait) {
-            display: inline;
-            padding: 40px 0;
-            color: $color--default;
-        }
-
-        @include media-query(tablet-landscape) {
-            padding: 30px 0;
-        }
-
-        &--active {
+        @include media-query(md) {
             &::after {
                 position: absolute;
                 inset-block-end: 0;
```

Comment:
I think I lean towards removing the `&--active` styling completely with this new design. With that I think we could completely remove this scss file and only use tailwind.